helm/chart-operator/values.yaml

```diff
@@ -10,7 +10,7 @@ helm:
   http:
     clientTimeout: "5s"
   kubernetes:
-    waitTimeout: "10s"
+    waitTimeout: "120s"
```

Comment:
10s is very short for charts with hooks. So I think its better to use the same default we have in the catalog. WDYT?

Reply:
Sure, Let's go with 120s.